src/KubeLibrary/KubeLibrary.py

```diff
@@ -58,6 +58,18 @@ def __init__(self, kube_config=None, incluster=False, cert_validation=True):
         if not cert_validation:
             self.v1.api_client.rest_client.pool_manager.connection_pool_kw['cert_reqs'] = ssl.CERT_NONE
 
+    def reload_config(self, kube_config=None, incluster=False, cert_validation=True):
+        """Reload the KubeLibrary to be configured with different optional arguments.
+           This can be used to connect to a different cluster during the same test.
+        - ``kube_config``:
+          Path pointing to kubeconfig of target Kubernetes cluster.
+        - ``incuster``:
+          Default False. Indicates if used from within k8s cluster. Overrides kubeconfig.
+        - ``cert_validation``:
+          Default True. Can be set to False for self-signed certificates.
+        """
+        self.__init__(kube_config=kube_config, incluster=incluster, cert_validation=cert_validation)
```

Comment:
I think it is safer to have some refugular function here, instead of __init__. Like:
```
def __init__(...)
  reload_config()

def reload_config(...):
   <content of current init>
```

Reply:
ok, understood. I'll update it like this.